cpp/include/raft/spectral/kmeans.hpp

```diff
@@ -31,10 +31,10 @@
 
 #include <raft/cudart_utils.h>
 #include <raft/linalg/cublas_wrappers.h>
+#include <raft/device_atomics.cuh>
 #include <raft/handle.hpp>
 #include <raft/spectral/matrix_wrappers.hpp>
 #include <raft/spectral/warn_dbg.hpp>
-#include <raft/utils/sm_utils.hpp>
```

Comment:
Perhaps a good thing that `sm_utils.hpp` is going away, but just as a CAVEAT, this might break dependencies in `cugraph` (possibly `cuml`, too).

Reply:
Thanks, for the cuGraph part, https://github.com/rapidsai/cugraph/pull/1021 will address the issue. AFAIK (I have searched for this), cuML does not use sm_utils.h

Reply:
Thanks, for cuGraph, https://github.com/rapidsai/cugraph/pull/1021 will address the issue. When I searched cuML, I didn't find any file including sm_utils.hpp from RAFT (Hope cuML folks will correct me if I am wrong).